xbmc/utils/JobManager.cpp

```diff
@@ -380,7 +380,10 @@ CJob *CJobManager::GetNextJob(const CJobWorker *worker)
   if (job)
     return job;
   // have no jobs
-  RemoveWorker(worker);
```

Comment:
hmmm, then there's need to pass in a `CJobWorker*` at all ?

Reply:
You're right. I pushed a fixup that gets rid of the `CJobWorker` entirely.